cron.go

```diff
@@ -43,31 +48,10 @@ type Job struct {
 	Name string
 	// Cron-formatted rhythm (ie. 0,10,30 1-5 0 * * *)
 	Rhythm string
-	// Routine method
-	Func func(context.Context) error
-
-	Repeats  int32
-	DueTime  string
```

Comment:
I don't want to add this here until we add support for each one of these. Keeping the original Rhythm only for now.

Reply:
We will want to keep this in mind then bc I believe this exists and is sent to the `ScheduleJob` so it'll error during our testing once your PR is merged

Reply:
Adding it here is not enough if the cron scheduler does not handle it.

Reply:
[I'm already passing it along](https://github.com/cicoyle/dapr/blob/feat-dist-scheduler/pkg/scheduler/server/api.go#L44-L47), so if I update the scheduler to use these changes it will break and I'll have to comment out those lines in the scheduler. I mean, its def not production code, but it's still partly there and being referenced and sent along. If you want I can comment them out. Just seems like extra orchestration